pkg/cli/admin/release/git.go

```diff
@@ -245,6 +246,13 @@ func mergeLogForRepo(g gitInterface, repo string, from, to string) ([]MergeCommi
 			mergeCommit.Subject = "Merge"
 		}
 
+		// Clean up the string so backticks in pr titles don't get formatted
+		// in the markdown output
+		mergeCommit.Subject = strings.Replace(mergeCommit.Subject, "`", "\\`", -1)
```

Comment:
Don't we want that?  I use this all the time:

> Bug 30498: Improve output of `oc status`

Reply:
Or are you concerned about unbalanced backticks?

Reply:
I believe when I found this a long time ago I found unbalanced backticks.  But right now i'd be happy just with the html escape. So I dropped this line.